project/settings/production.py

```
@@ -54,4 +64,8 @@
     },
 }
 
-ACCOUNT_EMAIL_VERIFICATION = "none"
+RESEND_API_KEY = config("RESEND_API_KEY")
+EMAIL_BACKEND = 'utils.email_backends.ResendEmailBackend'
+DEFAULT_FROM_EMAIL = config("RESEND_FROM_EMAIL")
+
+ACCOUNT_EMAIL_VERIFICATION = "mandatory"
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)